### ISSUE TITLE
SqlServerDsc: Fix minor review comments

### ISSUE THREAD

**Issue by johlju** [dsccommunity/SqlServerDsc#1881]
> Reviewed 10 of 10 files at r1, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/dsccommunity/SqlServerDsc/1877#-:-NRNyi_b5TdbgqMnmZXU:bnfp4nl)* status: :shipit: complete! all files reviewed, all discussions resolved (waiting on @johlju)

___
*[`azure-pipelines.yml` line 194 at r1](https://reviewable.io/reviews/dsccommunity/SqlServerDsc/1877#-NRNxigp--vNWTw8vm-3:-NRNxigp--vNWTw8vm-4:b-8n6vrs) ([raw file](https://github.com/dsccommunity/SqlServerDsc/blob/1e07bc46ecc3efb9f5b7594a2d1cbee5b4899ea6/azure-pipelines.yml#L194)):*
> ```YAML
>               Remove-Module -Name CommonTestHelper
>             name: removeSqlServerModule
>             displayName: 'Remove SqlServer module'
> ```

Can be fixed in another PR.

_Suggestion:_
```YAML
'Clean CI worker'
```
___
*[`azure-pipelines.yml` line 296 at r1](https://reviewable.io/reviews/dsccommunity/SqlServerDsc/1877#-NRNxuD91knXHefudn-y:-NRNxuD91knXHefudn-z:b-8n6vrs) ([raw file](https://github.com/dsccommunity/SqlServerDsc/blob/1e07bc46ecc3efb9f5b7594a2d1cbee5b4899ea6/azure-pipelines.yml#L296)):*
> ```YAML
>               Remove-Module -Name CommonTestHelper
>             name: removeSqlServerModule
>             displayName: 'Remove SqlServer module'
> ```

Can be fixed in another PR.

_Suggestion:_
```YAML
'Clean CI worker'
```
___
*[`source/Public/Import-SqlDscPreferredModule.ps1` line 113 at r1](https://reviewable.io/reviews/dsccommunity/SqlServerDsc/1877#-NRNy4ovA29KzeQYl8Ag:-NRNy4ovA29KzeQYl8Ah:b-8n6vrs) ([raw file](https://github.com/dsccommunity/SqlServerDsc/blob/1e07bc46ecc3efb9f5b7594a2d1cbee5b4899ea6/source/Public/Import-SqlDscPreferredModule.ps1#L113)):*
> ```PowerShell
>             $modulePathMachine = [System.String] [System.Environment]::GetEnvironmentVariable('PSModulePath', 'Machine')
> 
>             $modulePath = $modulePathSession + ';'  + $modulePathUser + ';' + $modulePathMachine
> ```

Can be fixed in another PR.

_Suggestion:_
```PowerShell
$modulePath = $modulePathSession, $modulePathUser, $modulePathMachine -join ';'
```

_Originally posted by @johlju in https://github.com/dsccommunity/SqlServerDsc/pull/1877#pullrequestreview-1357839167_
